library/core/src/pin.rs

```diff
@@ -909,3 +909,242 @@ impl<P, U> CoerceUnsized<Pin<U>> for Pin<P> where P: CoerceUnsized<U> {}
 
 #[stable(feature = "pin", since = "1.33.0")]
 impl<P, U> DispatchFromDyn<Pin<U>> for Pin<P> where P: DispatchFromDyn<U> {}
+
+/// Constructs a <code>[Pin]<[&mut] T></code>, by pinning[^1] a `value: T` _locally_[^2]
+/// (≠ [in the heap][`Box::pin`]).
```

Comment:
```suggestion
/// (≠ [on the heap][`Box::pin`]).
```
Normally things are put _on_ the heap (similar to _on_ the stack)

Reply:
Thanks! Feel free to call out other such instances, if you find any: I'm not a native speaker, so I can easily get some of these prepositions wrong 😅